job-server-extras/src/spark.jobserver/NamedBroadcast.scala

```diff
@@ -0,0 +1,32 @@
+package spark.jobserver
+
+import org.apache.spark.broadcast.Broadcast
+
+/**
+ * wrapper for named objects of type Broadcast
+ */
+case class NamedBroadcast[T](broadcast: Broadcast[T]) extends NamedObject
+
+/**
+  * implementation of a NamedObjectPersister for Broadcast objects
+  */
+class BroadcastPersister[T] extends NamedObjectPersister[NamedBroadcast[T]] {
+  override def persist(namedObj: NamedBroadcast[T], name: String) {
+    namedObj match {
+      case NamedBroadcast(broadcast) =>{}
```

Comment:
You could leave out the `{}`.  Actually, you could just make the entire body of this method empty, the match and case isn't doing anything.